src/addons/par/icc/tools/icc_common.sh

```diff
@@ -58,6 +58,11 @@ done
 
 set -ex
 
+if [[ "$script_dir" == "" ]]; then
+    echo "script_dir not set"
+    exit 1
+fi
+
```

Comment:
The script_dir is not a PATH, but rather the path to the PLSI ICC scripts (typically src/addons/par/icc/tools). I've added comments to document this.